project-generator/src/main/java/com/redhat/ipaas/project/converter/ProjectGenerator.java

```diff
@@ -15,13 +15,18 @@
  */
 package com.redhat.ipaas.project.converter;
 
+import com.redhat.ipaas.model.connection.Connector;
 import com.redhat.ipaas.model.integration.Integration;
 
 import java.io.IOException;
 import java.util.Map;
 
 public interface ProjectGenerator {
 
-    Map<String, byte[]> convert(Integration integration) throws IOException;
+    Map<String, byte[]> generate(Integration integration) throws IOException;
+
+    byte[] generatePom(Integration integration) throws IOException;
+
+    byte[] generatePom(Connector connector) throws IOException;
```

Comment:
wouldn't overload but give it a more descriptive name like 'generatePomForVerifierAction()' or so